### DISCUSSION TITLE
cannot validate certificate

### DISCUSSION THREAD

**Discussion by njavilas** [minio/minio#12847] [Q&A] [answered]
good evening I try to start minIO with the command server / data --console-address ": 443"

and environment variables

MINIO_ROOT_USER = fMhTRP2Jv6dsf41shU8eB
MINIO_ROOT_PASSWORD = v6hU84sd4f1eBfMhTRP2J
MINIO_DOMAIN = https: //s3.insertmendoza.com.ar
MINIO_BROWSER_REDIRECT_URL = https: //s3.insertmendoza.com.ar

and I get this error in console

debugging error: Post "https://172.19.0.3:9000/": x509: cannot validate certificate for 172.19.0.3 because it doesn't contain any IP SANs

**Comment by harshavardhana** [comment, marked as answer]
Make sure to set 

> MINIO_SERVER_URL=https://s3.insertmendoza.com.ar:9000

This is a wrong domain it's always `s3.insertmendoza.com.ar` - there is no `https://` in it.

https://github.com/minio/minio#things-to-consider please read documentation, do not know why you wouldn't read it.

**Reply by njavilas** [reply]
thanks for your quick response
I have read the documentation and it says in the example clearly
For example: export MINIO_SERVER_URL="https://minio.example.net"

when removing the https: // I have this error when executing it

ERROR Invalid MINIO_SERVER_URL value in environment variable: unexpected scheme found

**Reply by harshavardhana** [reply]
you cannot remove scheme https://github.com/minio/minio#things-to-consider you should read this

**Reply by njavilas** [reply]
my docker-compose it work! thanks!!
```
` minio:
    image: minio/minio
    restart: always
    
    ports: 
      - 443:443
      - 9001:9001
      - 9000:9000
```

    command: server /data  --address ":443" --console-address ":9001"

    volumes:
      - ./cdn/storage/:/data
      - ../cdn/letsencrypt/live/npm-14/fullchain.pem:/root/.minio/certs/public.crt
      - ../cdn/letsencrypt/live/npm-14/privkey.pem:/root/.minio/certs/private.key

    environment: 
      MINIO_ROOT_USER: 5sd456fsd42fsd42
      MINIO_ROOT_PASSWORD: 45sdf4sd5f15
      MINIO_SERVER_URL: https://s3.domain.com

    `
